kqueen_ui/blueprints/ui/views.py

```diff
@@ -555,12 +555,17 @@ class ClusterDelete(KQueenView):
 
     def handle(self, cluster_id):
         cluster = self.kqueen_request('cluster', 'get', fnargs=(cluster_id,))
+        msg = 'Cluster {} successfully deleted.'.format(cluster['name'])
+
```

Comment:
In this case there is really not a way around explicitly testing if we are on Manual engine, so I guess this is OK

Reply:
done